README.md

```diff
@@ -1,11 +1,17 @@
 # What is multipass?
 
-It's a system that orchestrates the creation, management and maintenance
-of virtual machines and associated Ubuntu images to simplify development.
+Multipass is a lightweight VM manager for Linux, Windows and macOS. It's designed
```

Comment:
Capitalize the 'm' for 'MacOS'.

Reply:
[No](https://www.apple.com/macos/).

Reply:
Ok, I stand corrected :smiley: I was basing that on the place where it _was_ `MacOS` (that you've since fixed).